resources/public/api/conf/3.0/schemas/broughtForwardLosses/listBFLoss/list_response.json

```diff
@@ -11,31 +11,69 @@
         "description": "Array of losses. An empty array will never be returned. In the instance that no losses exist for the supplied NINO, a MATCHING_RESOURCE_NOT_FOUND error will be returned.",
         "properties": {
           "lossId": {
-            "description": "The unique identifier of the loss",
+            "description": "The unique identifier of the loss.",
             "type": "string",
             "pattern": "^[A-Za-z0-9]{15}$",
             "example": "1234568790ABCDE"
           },
+          "businessId": {
+            "description": "An identifier for the business, unique to the customer.",
+            "type": "string",
+            "pattern": "^X[A-Z0-9]{1}IS[0-9]{11}$",
+            "example": "XPIS12345678910"
+          },
+          "typeOfLoss": {
+            "description": "The type of business the loss applies to.",
+            "type": "string",
+            "enum": [
+              "self-employment",
+              "self-employment-class4",
+              "uk-property-fhl",
+              "uk-property-non-fhl",
+              "foreign-property-fhl-eea",
+              "foreign-property"
+            ]
+          },
+          "lossAmount": {
+            "description": "Brought Forward Loss amount related to the type of loss specified. The value must be between 0 and 99999999999.99 up to 2 decimal places.",
+            "type": "number",
+            "multipleOf": 0.01,
+            "minimum": 0.00,
+            "maximum": 99999999999.99,
+            "example": "5000.99"
+          },
+          "taxYearBroughtForwardFrom": {
+            "description": "The tax year to which the loss applies. The start year and end year must not span two tax years. No gaps are allowed, for example, 2020-22 is not valid. The minimum tax year is 2018-19. The tax year must have ended.",
+            "type": "string",
+            "example": "2022-23"
+          },
+          "lastModified": {
+            "description": "The timestamp when the entry was created or last modified.",
+            "type": "string",
+            "format": "date-time",
+            "pattern": "^2[0-9]{3}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}(\\.[0-9]{0,3}|)Z$",
+            "example": "2021-07-13T12:13:48.763Z"
+          },
           "links": {
             "type": "array",
             "items": {
               "type": "object",
-              "description": "A list of endpoint links that indicate possible actions related to the current resource",
+              "description": "A list of endpoint links that indicate possible actions related to the current resource.",
```

Comment:
The links are related to the item in the list not the resource as a whole. What do we normally say here?

Reply:
Yh, but I'm guessing this was adopted other apis for example [here](https://github.com/hmrc/self-assessment-accounts-api/blob/main/resources/public/api/conf/1.0/schemas/list_charges_response.json)